examples/webservice/java/src/main/java/es/nangel/logstash/lookup/webservice/test/Test.java

```diff
@@ -0,0 +1,11 @@
+package es.nangel.logstash.lookup.webservice.test;
```

Comment:
Are these tests? If so, can you move them to `/spec/unit` or `spec/integration` ?